src/microbe_stage/MicrobeStage.tscn

```diff
@@ -2215,7 +2223,6 @@ __meta__ = {
 }
 
 [node name="PauseMenu" parent="." instance=ExtResource( 11 )]
-visible = false
```

Comment:
Ditto.

Reply:
Actually this is the expected result. The pause menu is hidden by default, which means Godot should not write the visible property at all (as it is the default value). I recently accepted a PR that added this explicit `visible = false` here, but it seems that Godot will just remove that automatically. So I think we should just keep this change.